index.html

```diff
@@ -0,0 +1,342 @@
+         <li class="tonic dot"></li>
+         <li class="tonic year">2015</li>
+       </ul>
+
+       <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     
+       <ul class="resources">
+        <li><a  class="button" href="#">HTML</a> <li>
+        <li><a  class="button" href="#">CSS</a> </li>
+        <li><a  class="button" href="#">javaScript</a></li>      
+      </ul>
+
+        <div class="action"> 
+          <form class="normalbutton">
+            <a class="seeproject"  href="#">See Project</a> 
+          </form> 
+      </div> 
+      
+
+      
+      </div>
+    </div>
+
+    <!-- <div class="card">
+      <img src="img_avatar.png" alt="Avatar" style="width:100%">
+      <div class="container">
+        <h4><b>John Doe</b></h4>
+        <p>Architect & Engineer</p>
+      </div>
+    </div>
+
+    <div class="card">
+      <img src="img_avatar.png" alt="Avatar" style="width:100%">
+      <div class="container">
+        <h4><b>John Doe</b></h4>
+        <p>Architect & Engineer</p>
+      </div>
+    </div> -->
+
+  <!--Page 3-->
+     <div class="card2">
+      <img id="avator" src="sources/Portfolio2.png" alt="Avatar">
+      <div class="container2">
+        <h2><b>Multi-Post</br>
+          Stories</b></h2>
+
+       <ul class="tonic">
+        <li><a class="tonic canopy" href="#">CANOPY</a></li>
+        <li class="tonic dot"></li>
+        <li class="tonic back">Back End Dev</li>
+         <li class="tonic dot"></li>
+         <li class="tonic year">2015</li>
+       </ul>
+
+       <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     
+       <ul class="resources">
+        <li><a  class="button" href="#">HTML</a> <li>
+        <li><a  class="button" href="#">CSS</a> </li>
+        <li><a  class="button" href="#">javaScript</a></li>      
+      </ul>
+
+        <div class="action"> 
+          <form class="normalbutton">
+            <a class="seeproject"  href="#">See Project</a> 
+          </form> 
+        </div> 
+      
+
+      
+      </div>
+    </div>
+
+    <!--page 4--> 
+    
+    <div class="card1">
+      <img id="avator" src="sources/Portfolio3.png" alt="Avatar">
+      <div class="container">
+        <h2><b>Tonic</b></h2>
+
+       <ul class="tonic">
+        <li><a class="tonic canopy" href="#">CANOPY</a></li>
+        <li class="tonic dot"></li>
+        <li class="tonic back">Back End Dev</li>
+         <li class="tonic dot"></li>
+         <li class="tonic year">2015</li>
+       </ul>
+
+       <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     
+       <ul class="resources">
+        <li><a  class="button" href="#">HTML</a> <li>
+        <li><a  class="button" href="#">CSS</a> </li>
+        <li><a  class="button" href="#">javaScript</a></li>      
+      </ul>
+
```

Comment:
- I think the padding on each list and font-size is a bit too big, compared to what's on figma. 🤗 

Can we reduce it?? 

Here is what i mean 👇🏽 
| Your List  |That of Figma |
| :---:   | :---: | 
| <img width="375" alt="Screenshot 2022-12-08 at 13 24 44" src="https://user-images.githubusercontent.com/99246359/206446884-c4f61eab-dfb5-4db8-92cf-e9d276a3ce19.png"> |!<img width="370" alt="Screenshot 2022-12-08 at 13 25 18" src="https://user-images.githubusercontent.com/99246359/206446954-18335887-9e20-43fa-a517-56bcff2a3b03.png">  |


⚠️ Kindly update this changes to all  list that applies.